src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -171,8 +172,8 @@ object GermlineAssemblyCaller {
 
       // Build a variant using the current offset and read evidence
       def buildVariant(referenceOffset: Int,
-                       referenceBases: Array[Byte],
```

Comment:
Yea I think this folds in to the `ContigSequence` discussion [above](https://github.com/hammerlab/guacamole/pull/417#discussion_r58218719).